remove orphaned packages

jeremyd2019 opened the pull request:
```
pacman -Rns $(pacman -Qmq)
```

automake1.10 (removed)
automake1.6 (removed)
automake1.7 (removed)
automake1.8 (removed)
automake1.9 (removed)
crypt (removed)
libcrypt (removed)

plus, remove unowned files from /usr/bin (apparently added temporarily to work around issues but then never removed again)

dscho commented:
Just like I did [in the sibling PR](https://github.com/git-for-windows/git-sdk-64/pull/97#pullrequestreview-2623370153), I kicked off [a `git-artifacts` run](https://github.com/dscho/git-sdk-64/actions/runs/13393411687) to verify that this can be safely merged.

dscho commented:
> Just like I did [in the sibling PR](https://github.com/git-for-windows/git-sdk-64/pull/97#pullrequestreview-2623370153), I kicked off [a `git-artifacts` run](https://github.com/dscho/git-sdk-64/actions/runs/13393411687) to verify that this can be safely merged.

This run succeeded! 🎉 

@jeremyd2019 I opened a companion PR over in git-sdk-arm64 and used the opportunity to add [a commit to remove the unwanted `/mingw32/` and `/mingw64/` files, too](https://github.com/git-for-windows/git-sdk-arm64/pull/35/commits/be34796fb942f505e4691d2db04836ef4c5982a7). What do you think? Can we repeat that exercise in this here PR, too?

jeremyd2019 commented:
It looks like this might remove a few files that are actually worth keeping...
```
        deleted:    mingw32/etc/gitattributes
        deleted:    mingw32/etc/gitconfig
        deleted:    mingw32/etc/xml/catalog
        deleted:    mingw64/etc/gitattributes
        deleted:    mingw64/etc/gitconfig
        deleted:    mingw64/etc/xml/catalog
        deleted:    mingw64/share/man/man3/Git.3pm
        deleted:    mingw64/share/man/man3/Git.I18N.3pm
        deleted:    mingw64/share/man/man3/Git.SVN.Editor.3pm
        deleted:    mingw64/share/man/man3/Git.SVN.Fetcher.3pm
        deleted:    mingw64/share/man/man3/Git.SVN.Memoize.YAML.3pm
        deleted:    mingw64/share/man/man3/Git.SVN.Prompt.3pm
        deleted:    mingw64/share/man/man3/Git.SVN.Ra.3pm
        deleted:    mingw64/share/man/man3/Git.SVN.Utils.3pm
        deleted:    mingw64/var/cache/fontconfig/CACHEDIR.TAG
        deleted:    mingw64/var/cache/fontconfig/d031bbba323fd9e5b47e0ee5a0353f11-x86_64.cache-7
        deleted:    mingw64/var/cache/fontconfig/d031bbba323fd9e5b47e0ee5a0353f11-x86_64.cache-8
        deleted:    mingw64/var/cache/fontconfig/d031bbba323fd9e5b47e0ee5a0353f11-x86_64.cache-8.NEW
        deleted:    mingw64/var/cache/fontconfig/ef9c9ad8cc5857eb63cb3660bc8bd202-x86_64.cache-7.NEW
```

jeremyd2019 commented:
Actually libxml2 was removed in #97, so etc/xml/catalog can go.  I don't see any font stuff installed either, so the fontconfig cache can probably go too.  Just git config files?

dscho commented:
> It looks like this might remove a few files that are actually worth keeping here...
> 
> ```
>         deleted:    mingw32/etc/gitattributes
>         deleted:    mingw32/etc/gitconfig
>         deleted:    mingw64/etc/gitattributes
>         deleted:    mingw64/etc/gitconfig
> ```

Git for Windows moved it back to `/etc/gitconfig` (and upstreamed that change as git/git@fb5e3378f8427ef970544848974db2dc90828a7a). Therefore, these files are not even used; Their presence is misleading.

> ```
>         deleted:    mingw64/share/man/man3/Git.3pm
>         deleted:    mingw64/share/man/man3/Git.I18N.3pm
>         deleted:    mingw64/share/man/man3/Git.SVN.Editor.3pm
>         deleted:    mingw64/share/man/man3/Git.SVN.Fetcher.3pm
>         deleted:    mingw64/share/man/man3/Git.SVN.Memoize.YAML.3pm
>         deleted:    mingw64/share/man/man3/Git.SVN.Prompt.3pm
>         deleted:    mingw64/share/man/man3/Git.SVN.Ra.3pm
>         deleted:    mingw64/share/man/man3/Git.SVN.Utils.3pm
> ```

If you download https://github.com/git-for-windows/pacman-repo/releases/download/2025-02-13T12-51-22.000Z/mingw-w64-x86_64-git-doc-man-2.48.1.1.2bd190bcd2-1-any.pkg.tar.xz (a regular `git-sdk-64` checkout won't have those files, only `mingw-w64-git-doc-html` is installed, but not `mingw-w64-git-doc-man` because manual pages are somewhat at odds with a user-friendly experience on Windows, therefore Git for Windows does not ship them, only the HTML versions) you can verify that these files are not meant to be part of Git for Windows:

```console
$ tar tvf mingw-w64-x86_64-git-doc-man-2.48.1.1.2bd190bcd2-1-any.pkg.tar.xz |
   grep Git
-rw-r--r-- root/root      9633 2025-02-13 12:00 mingw64/share/man/man3/Git.3pm.gz
```

It's not only that the manual pages are unwanted, the non-`.gz`-ed versions are not even meant to be part of a Pacman package.

So yes, all of those files can go.

jeremyd2019 commented:
> > It looks like this might remove a few files that are actually worth keeping here...
> > ```
> >         deleted:    mingw32/etc/gitattributes
> >         deleted:    mingw32/etc/gitconfig
> >         deleted:    mingw64/etc/gitattributes
> >         deleted:    mingw64/etc/gitconfig
> > ```
> 
> Git for Windows moved it back to `/etc/gitconfig` (and upstreamed that change as [git/git@fb5e337](https://github.com/git/git/commit/fb5e3378f8427ef970544848974db2dc90828a7a)). Therefore, these files are not even used; Their presence is misleading.

Off-topic here, but BTW, why does arm64 git-for-windows seem to want them in clangarm64/etc?

dscho commented:
> why does arm64 git-for-windows seem to want them in clangarm64/etc?

That's a bug. I remember you mentioning this elsewhere and I meant to split it out into a separate ticket. Will do so now.

dscho commented:
> > why does arm64 git-for-windows seem to want them in clangarm64/etc?
> 
> That's a bug. I remember you mentioning this elsewhere and I meant to split it out into a separate ticket. Will do so now.

Done: https://github.com/git-for-windows/git/issues/5431

dscho commented:
I've started [another `git-artifacts` run](https://github.com/dscho/git-sdk-64/actions/runs/13399784758) with the latest version of this PR branch; It's getting late over here, I'll probably come back to this PR tomorrow (hopefully being delighted by a simply successful run).
